lib/galaxy/webapps/galaxy/api/users.py

```diff
@@ -823,7 +823,8 @@ def get_information(self, trans, id, **kwd):
             "username": username,
         }
         is_galaxy_app = trans.webapp.name == "galaxy"
-        if (trans.app.config.enable_account_interface and not trans.app.config.use_remote_user) or not is_galaxy_app:
+        allow_profile_edit = trans.app.config.enable_account_interface
```

Comment:
The old behaviour of not letting remote users edit profile should also be preserved I think, so we can just revert that last commit.
```suggestion
        allow_profile_edit = (
            trans.app.config.enable_account_interface
            and not trans.app.config.use_remote_user
            and not trans.app.config.disable_local_accounts
        )
```
I think what this PR was doing already was correct.